INSTALL.md

```diff
@@ -25,23 +18,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
     1. ANDROID_SDK_ROOT: Enter the directory from Step 1.4 above
     2. NDK_ROOT: C:\ndk\android-ndk-r12b
     3. ANT_ROOT: C:\ant\apache-ant-x.xx.x\bin
-## Build and install goa
-Goa is an application which has many games for a child to play while improving literacy.
+## Installation steps for all apps
 1. Make sure an Android device is connected with USB debug permissions set.
 2. Overwrite the directory *tools* in ANDROID_SDK_ROOT with C:\tools
 3. `cd goa`
 4. `cocos compile -p android`
 5. Copy the content of _proj.android\libs_ folder into _proj.android-studio\goa\src\main\jniLibs_ folder. __Note__: Copy only _.so_ files.
 6. Start Android Studio and import _proj.android-studio_ folder into it.
-7. Build and run the application.
+7. Download the OBB file from the latest GitHub release and follow the instructions provided there.
```

Comment:
Hyperlink this to point to [GitHub release](github.com/maqsoftware/Simple-Class/releases)